sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/catalog/SessionCatalog.scala

```diff
@@ -81,14 +90,27 @@ class SessionCatalog(
     if (conf.caseSensitiveAnalysis) name else name.toLowerCase
   }
 
+  private[sql] def makeQualifiedPath(path: String): Path = {
+    val hadoopPath = new Path(path)
+    makeQualifiedPath(hadoopPath)
+  }
+
+  private  def makeQualifiedPath(path: Path): Path = {
```

Comment:
btw my previous comment about javadoc explaining why we need to fully qualified path applies to this function too